### PR TITLE
Update get-tlsciphersuite.md

### PR DESCRIPTION
**Pull request by sukihanabi**
#1067 
Action Taken, added the ff:
- "The Output includes a Protocols field that represent the TLS/SSL protocol versions for which that cipher is available."
-Article
Supported Cipher Suites and Protocols in the Schannel SSP https://docs.microsoft.com/previous-versions/windows/it-pro/windows-server-2012-R2-and-2012/dn786419(v=ws.11)
Cipher Suites in TLS/SSL (Schannel SSP)
https://docs.microsoft.com/windows/desktop/secauthn/cipher-suites-in-schannel
----------------------------------------------------------------------
References:

TLS Cipher Suites in Windows 10 v1511
https://docs.microsoft.com/en-us/windows/desktop/secauthn/tls-cipher-suites-in-windows-10-v1511

TLS Cipher Suites
https://docs.microsoft.com/en-us/windows/desktop/secauthn/tls-cipher-suites

https://serverfault.com/questions/910177/what-is-the-meaning-of-the-values-of-the-protocols-field-from-get-tlsciphersuite

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: docset/windows/tls/get-tlsciphersuite.md [hunk 1]
```diff
@@ -108,6 +108,8 @@ Protocols             : {771, 65277}
 ```
 
 This command gets all the cipher suites that have names that contain the string AES.
+The output includes a protocols field that represent the TLS/SSL protocol versions for which that cipher is available. See [Supported Cipher Suites and Protocols in the Schannel SSP](https://docs.microsoft.com/previous-versions/windows/it-pro/windows-server-2012-R2-and-2012/dn786419(v=ws.11)) and [Cipher Suites in TLS/SSL (Schannel SSP)](https://docs.microsoft.com/windows/desktop/secauthn/cipher-suites-in-schannel) for more information.
```
Comment (nenonix) [inline docset/windows/tls/get-tlsciphersuite.md]:
```suggestion
The output includes a protocols field that represents the TLS/SSL protocol versions for which that cipher is available. See [Supported Cipher Suites and Protocols in the Schannel SSP](https://docs.microsoft.com/previous-versions/windows/it-pro/windows-server-2012-R2-and-2012/dn786419(v=ws.11)) and [Cipher Suites in TLS/SSL (Schannel SSP)](https://docs.microsoft.com/windows/desktop/secauthn/cipher-suites-in-schannel) for more information.
```

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

**Comment by get-itips** [comment]
Hello @sukihanabi 
I reviewed the links that you are providing in the edit, they are good, but I think we should add a link that explains the Procols version that we see when running the cmdlet, the problem is I could not find an official article about it. For example 771 means TLSv1.2 and 65277 es DTLSv1.2 but could not find official documentation to use that as a link.
What do you think guys @dariomws306  @tseward?

**Comment by dariomws306** [comment]
@get-itips, yes I am agree. User is asking for more information about "Protocols". And I cannot find official documentation about them neither.
Can we validate with PT if these are correct:
    # SSL
    0x0002: "SSL_2_0",
    0x0300: "SSL_3_0",
    # TLS:
    0x0301: "TLS_1_0",
    0x0302: "TLS_1_1",
    0x0303: "TLS_1_2",
    0x0304: "TLS_1_3",
    # DTLS
    0x0100: "PROTOCOL_DTLS_1_0_OPENSSL_PRE_0_9_8f",
    0x7f10: "TLS_1_3_DRAFT_16",
    0x7f12: "TLS_1_3_DRAFT_18",
    0xfeff: "DTLS_1_0",
    0xfefd: "DTLS_1_1",
There are more articles with this information like (RFC so they are "official"):
https://tools.ietf.org/id/draft-ietf-tls-tls13-28.html
But I can find the DTLS version RFC.

Also may we can include 
https://docs.microsoft.com/en-us/windows/desktop/secauthn/cipher-suites-in-schannel
But this should be remove because it's and old article:
https://docs.microsoft.com/previous-versions/windows/it-pro/windows-server-2012-R2-and-2012/dn786419(v=ws.11)

**Comment by sukihanabi** [comment]
@get-itips & @dariomws306 
Yes, I agree . I can't neither find official article for protocol version.
I have found this information in GitHub article; would this help.
https://github.com/tintinweb/scapy-ssl_tls/blob/eb6df1c940527e65e3f16c662836d7382ba4ace0/scapy_ssl_tls/ssl_tls.py#L241

Or perhaps when the above protocol version has been verified we can add it in the article.

Any advise to resolve the case is much appreciated. Thanks!

**Comment by get-itips** [comment]
@sukihanabi I found the full list of constants [here](https://docs.zeek.org/en/stable/scripts/base/protocols/ssl/consts.bro.html) but we can not link to an unofficial resource.

We need either to contact ssl product team or @kenwith  on this one. imho.

**Comment by sukihanabi** [comment]
@get-itips I see, we will wait then for our next step. thank you for the help!

**Comment by get-itips** [comment]
> @get-itips I see, we will wait then for our next step. thank you for the help!

Yes, but your Project Coordinator has the last word on this.

**Comment by tseward** [comment]
They are documented at https://docs.microsoft.com/en-us/windows/desktop/api/bcrypt/nf-bcrypt-bcryptderivekey, but that's about the only place I can find that is 'official'. It's missing TLS 1.3/DTLS_1.1.

**Comment by dariomws306** [comment]
Thank you @tseward. I think it's our better option.

Second could be add a table but I prefer Trevor link.

**Comment by sukihanabi** [comment]
> They are documented at https://docs.microsoft.com/en-us/windows/desktop/api/bcrypt/nf-bcrypt-bcryptderivekey, but that's about the only place I can find that is 'official'. It's missing TLS 1.3/DTLS_1.1.

Thank you @tseward & @dariomws306 , added the article as suggested. feel free to let me know for any changes/suggestion.

**Comment by dariomws306** [comment]
Looks good.

Maybe these links could be moved to the description of the cmdlet.

**Comment by sukihanabi** [comment]
> Looks good.
> Maybe these links could be moved to the description of the cmdlet.

-change has been made, fee free to check. Thank you!

**Comment by get-itips** [comment]
Looks good

**Comment by sukihanabi** [comment]
> Looks good

Thank you, appreciate all the help.

**Comment by dariomws306** [comment]
I am ok now.

**Comment by e0i** [comment]
The full list of protocol constants will be dealt with separately. Thank you for the productive discussion.

**Comment by e0i** [comment]
@AndreaBarr The PR can be merged. Thanks.

**Pull request merged**